build/windows/omsupply_server.suf

```diff
@@ -4276,6 +4279,21 @@ end
 
 File.Delete(sFileToBeDeletePath); -- Delete unchoosen app file (.exe) by a user
 
+-- Add a firewall rule to allow incoming requests
+local firewallRuleKey = "SYSTEM\\CurrentControlSet\\Services\\SharedAccess\\Parameters\\FirewallPolicy\\FirewallRules";
+if (Registry.DoesKeyExist(HKEY_LOCAL_MACHINE, firewallRuleKey)) then
+
+	Registry.SetValue(HKEY_LOCAL_MACHINE, firewallRuleKey, "open-mSupply-Server-TCP-IN", "v2.28|Action=Allow|Active=TRUE|Dir=In|Protocol=6|LPort=" .. SessionVar.Get("%omsupply_port%") .. "|Name=omSupply Server|", REG_SZ);

```

Comment:
Hmm shouldn't Name also come from SessionVar as ProductName

Reply:
updated with f13595469

Reply:
> it's just a firewall rule so I didn't bring that through

Ahh ok, i saw that it's been done for port `.. SessionVar.Get("%omsupply_port%") ..`

> updated with https://github.com/openmsupply/open-msupply/commit/f135954691e6d179b6ad1b3aa797c83d07fcf4f3

Thanks